roosevelt.js

```diff
@@ -62,6 +64,10 @@ module.exports = function (params) {
 
   logger.log('💭', `Starting ${appName} in ${appEnv} mode...`.bold)
 
+  haveStale = staleCheck()
```

Comment:
To simplify this workflow a bit how about moving all of the functionality of the check into the buildScanner module (Including the logging part). Then all roosevelt.js needs to do is call it.